### PR TITLE
feat: added modal to handle confirm logout

### LINKED ISSUE #223: [Frontend] logout window (desktop)
🚨 **Attention: Please read the guidelines before applying; otherwise, you will be ignored.** 🚫📋
0. Please, join the telegram group to keep in touch https://t.me/+m0IGRiBvLtsyMzIy 
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Add pop up window when user wants to log out
3. If user `click` on disconnet, disconnet its wallet
4. If user `click` on `cancel` close this pop-up window
5. Ask me in telegram(djeck_vorobey1) to get access to figma, send me your email in message

### PR DESCRIPTION
**Pull request by Ayoazeez26**
This PR fixes #223

### CODE REVIEW

File: frontend/src/components/Logout/logoutModal.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  border: none;
+  color: white;
+}
+
+
+.card-value {
+  font-size: 24px;
+  font-weight: 600;
+  color: white;
+}
```
Comment (whateverfw) [inline frontend/src/components/Logout/logoutModal.css:54]: seems like this clases is not relavant to logout modal

File: frontend/src/components/Logout/logoutModal.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+.main-card {
+  background: rgba(18, 7, 33, 1);
+  border: 1px solid rgba(54, 41, 78, 1);
```
Comment (whateverfw) [inline frontend/src/components/Logout/logoutModal.css:58]: use css variables if already exists, if not create new ones (check global.css and figma hex colors)

File: frontend/src/components/Logout/logoutModal.css [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  width: 642px;
+  height: 318px;
+  padding: 16px 24px 16px 24px;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.logout-heading-container {
+  border-bottom: 1px solid #36294E;
```
Comment (whateverfw) [inline frontend/src/components/Logout/logoutModal.css:70]: use css variable

File: frontend/src/components/Logout/logoutModal.css [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  display: flex;
+  gap: 24px;
+  width: 100%;
+}
+
+.logout-btn {
+  width: 642px;
+  height: 60px;
+  border-radius: 900px;
+  background: linear-gradient(73.48deg, #e01dee 1.13%, #49abd2 51.27%);
```
Comment (whateverfw) [inline frontend/src/components/Logout/logoutModal.css:98]: use css variable

File: frontend/src/components/Logout/logoutModal.css [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+.cancel-button::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border-radius: 50px;
+  padding: 2px;
+  background: linear-gradient(90deg, #74D6FD 0%, #E01DEE 100%);
```
Comment (whateverfw) [inline frontend/src/components/Logout/logoutModal.css:132]: use css variable

File: frontend/src/components/Logout/logoutModal.css (hunk 4 above)
Comment (whateverfw) [inline frontend/src/components/Logout/logoutModal.css:98]: use css variables

File: frontend/src/components/Logout/LogoutModal.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      <div className="main-container">
+        <div className="main-card">
+          <div className="logout-heading-container">
+            <p className="logout-heading">Logout</p>
+          </div>
+
+          <p className="logout-text">Do you want to disconnect your wallet and logout of this account?</p>
+        </div>
+        <div className="buttons-container">
+          <button className="cancel-button" onClick={() => onClose()}>
```
Comment (whateverfw) [inline frontend/src/components/Logout/LogoutModal.jsx:18]: can be just onClick={onClose}

File: frontend/src/components/Logout/LogoutModal.jsx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+            Cancel
+          </button>
+          <button
+            className="logout-btn"
+            onClick={() => onLogout()}
```
Comment (whateverfw) [inline frontend/src/components/Logout/LogoutModal.jsx:23]: onClick={onLogout}

File: frontend/src/components/Logout/LogoutModal.jsx [hunk 8]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./logoutModal.css";
+
+export default function LogoutModal({ onClose, onLogout }) {
+
+
+  return (
```
Comment (whateverfw) [inline frontend/src/components/Logout/LogoutModal.jsx:7]: remove redundant spacing

File: frontend/src/App.jsx [hunk 9]
```diff
@@ -70,19 +73,30 @@ function App() {
   const handleLogout = () => {
     logout();
     setWalletId(null);
+    setShowModal(false);
```
Comment (whateverfw) [inline frontend/src/App.jsx:76]: use closeModal function that already has closing functionality

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Comment by Ayoazeez26** [comment]
Comments addressed @whateverfw

**Review by whateverfw — CHANGES_REQUESTED** [review]
image
Cancel button has broken UI with squared corners, please fix

**Comment by djeck1432** [comment]
@Ayoazeez26 any updates?

**Review by djeck1432 — APPROVED** [review]
LGTM

**Pull request merged**